src/common/navigation.yaml

```diff
@@ -130,8 +130,8 @@ sections:
               - path: /language-keywords
               - path: /language-functions
 
-  - title: Ecosystem
-    pages:
-      - path: /overview #pbc et al
-      - path: /stacks-token
-      - path: /contributing
+      - path: /ecosystem
+        pages:
+          - path: /overview #pbc et al
+          - path: /stacks-token
```

Comment:
I also wonder how relevant content like the token details are 🤔 do we know from analytics how often this page is used? looks like there's no other page cover it outside of the docs (in that detail), so maybe we should keep it for now.

Reply:
Fathom seems to have broken in some way in late April but looking at historical analytics the token details is not a hugely viewed page - it's way down the list from pages that contain actual instructional material like the stacking page and tutorials.